README.md

```diff
@@ -26,7 +27,7 @@ MicroProfile Fault Tolerance, adding application-specific capabilities such as f
 
 ## Included Components
 - [MicroProfile](https://microprofile.io)
-- [Istio](https://istio.io/)
+- [Istio (1.6)](https://istio.io/)
```

Comment:
Why are we making this change? There is no information in any of the commit to explain it?

Reply:
I add the version number because this pattern's policy rule only works in Istio 1.6. Istio 2.x has a different format for policy rule.

Reply:
Should we change the link to point to the specific version, or is the link okay as is?